packages/xstate-wallet/src/store/store.ts

```diff
@@ -18,14 +18,16 @@ export interface Store {
     appDefinition?: string
   ): Promise<ChannelStoreEntry>;
   getEntry(channelId): Promise<ChannelStoreEntry>;
+
+  ledgerFeed: Observable<LedgerUpdated>;
+  lockLedger(participantId: string): Promise<LedgerUpdated>;
```

Comment:
If we ever wanted to allow for replace-by-incentive apps in the wallet, we'll also need some sort of lock mechanism on app channels I think. Should we implement a more general locking API on a channel ID instead of specifically for ledger channels?

Reply:
That might be a good idea; it's probably worth separating the `ledgerId` lookup from the ledger channel's lock status